test/unit/rules/resources/route53/test_recordset_name.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
```suggestion
Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

[can remove this now](https://github.com/aws-cloudformation/cfn-python-lint/pull/1528)